README.md

```diff
@@ -1,70 +1,133 @@
+- [ ] I installed linter react&redux linters
+- [ ] I used Gitflow.
+- [ ] I made sure that you documented your work in a professional way.
+- [ ] Follow our list of best practices for HTML, CSS and JavaScript.
+- [ ] I used React components.
+- [ ] I used React props.
+- [ ] I used React Router.
+- [ ] I connected React and Redux.
+- [ ] I used store, actions and reducers in React.
+- [ ] I applied React best practices and language style guides in code.
+- [ ] I wrote integration tests with a React testing library.
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+<!-- Features -->
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+### Key Features <a name="key-features">
 
-## Learn More
+- This project, built React and Redux.
+- Mobile Version.
+- Single-page application (SPA)
+- The data retrieved from the API
+  </a>
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+<!-- LIVE DEMO -->
 
-### Code Splitting
+## 🚀 Live Demo <a name="live-demo"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+- [Live Demo Link](https://tx-breweries.onrender.com)
+- [Video Demo](https://www.loom.com/share/99a54ff913bc4fca93fec0f5a4f7cf60)
 
-### Analyzing the Bundle Size
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+<!-- GETTING STARTED -->
 
-### Making a Progressive Web App
+## 💻 Getting Started <a name="getting-started"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+To get a local copy up and running,clone this repository to your desired folder: git@github.com:rushemaa/metrics-webapp.git
```

Comment:
- [optional] kindly add more information in the getting started section on how to go about cloning your app till seeing the final product.